build.gradle

```diff
@@ -44,6 +44,8 @@ dependencies {
 	//implementation 'org.apache.maven:maven-aether-provider:3.0.5'
 	//implementation 'com.google.guava:guava:28.2-jre'
 
+	implementation 'org.jetbrains.kotlin:kotlin-compiler:1.5.0'
```

Comment:
Do we need these when we are downloading the kotlin tools ?

Im very eery on growing the size of jbang itself if we can avoid it. 

This change moves it from 5.9mb to 7.7mb ... can it be avoided ?

Reply:
looks like the *only* thing these are used for is @NotNull and a default string version for kotlin. That isn't justifiable; but also super easy to fix by just removing them.

Reply:
yeah.   old code and gone now.